src/replays/replays.worker.ts

```diff
@@ -57,12 +58,16 @@ const methods: WorkerSpec = {
       throw new Error("Game settings could not be properly loaded.");
     }
 
-    if (settings.players.length !== 2) {
+    if (settings.players.length !== 2 && settings.gameMode != GameMode.TARGET_TEST) {
```

Comment:
Why are we changing this condition? Wouldn't a target test game still return `game.getStats()` which doesn't really make sense. Shouldn't we do a proper game mode check to ensure that we're calculating stats for a mode we actually support?